front/src/reducers/osrdconf/stdcmConf/index.ts

```diff
@@ -1,11 +1,12 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { OsrdStdcmConfState } from 'applications/operationalStudies/consts';
 import { Draft } from 'immer';
-import { defaultCommonConf, buildCommonConfReducers } from '../common';
+import { defaultCommonConf, buildCommonConfReducers } from '../osrdConfCommon';
 
 export const stdcmConfInitialState: OsrdStdcmConfState = {
   maximumRunTime: 43200,
   standardStdcmAllowance: undefined,
+  stdcmMode: 'standard',
```

Comment:
stdcmMode is not used anymore, so you can remove it and everything which is related to it (selector, action...)

Reply:
Done